tensorboard/plugins/graph/tf_graph_common/node.ts

```diff
@@ -1366,7 +1361,7 @@ export function buildGroupForAnnotation(
     .each(function (a) {
       let aGroup = d3.select(this);
```

Comment:
Can you remove aGroup's declaration, too?

Reply:
Thanks, done.